helm-charts/azure-api-management-gateway/README.md

```diff
@@ -109,6 +109,7 @@ their default values.
 | `observability.statsD.sampling` | Defines the metrics sampling rate. Value can be between `0` and `1`. e.g., `0.5`. Learn more in [our documentation](https://docs.microsoft.com/azure/api-management/how-to-configure-local-metrics-logs#configure-the-self-hosted-gateway-to-emit-metrics). | 1 |
 | `observability.statsD.tagFormat` | Defines the [tagging format](https://github.com/prometheus/statsd_exporter#tagging-extensions) in StatsD metrics as per the official docs. Value can be `none`, `librato`, `dogStatsD`, `influxDB`. Learn more in [our documentation](https://docs.microsoft.com/azure/api-management/how-to-configure-local-metrics-logs#configure-the-self-hosted-gateway-to-emit-metrics). | N/A |
 | `service.type` | Type of Kubernetes service to use to expose to serve traffic | `ClusterIP` |
+| `service.loadBalancerIP` | Attach a pre-existing static IP to a `LoadBalancer` type service. Learn more in the [Kubernetes docs](https://kubernetes.io/docs/concepts/services-networking/service/#loadbalancer). | |
```

Comment:
Nit - Let's do this in case we want to add more flags in the future
```suggestion
| `service.loadBalancer.ip` | Attach a pre-existing static IP to a `LoadBalancer` type service. Learn more in the [Kubernetes docs](https://kubernetes.io/docs/concepts/services-networking/service/#loadbalancer). | |
```

Reply:
Change added.